frontend2/src/views/ErrorBoundary.tsx

```diff
@@ -1,26 +1,37 @@
 import React from "react";
-import { useRouteError } from "react-router-dom";
+import { useRouteError, Link } from "react-router-dom";
 import { PageTitle } from "../components/elements/BattlecodeStyle";
+import { useEpisodeId } from "../contexts/EpisodeContext";
 
 const ErrorBoundary: React.FC = () => {
   const error = useRouteError();
+  const { episodeId } = useEpisodeId();
 
-  if (error instanceof Error) {
-    return (
-      <div className="flex flex-col gap-6 p-6 xl:flex-row">
+  return (
+    <div className="flex h-screen w-full items-center justify-center bg-white p-6">
+      <div className="flex w-full max-w-xs flex-col gap-4 rounded-lg border border-gray-300 bg-gray-50 p-6 shadow-md">
         <PageTitle>Oops, something went wrong.</PageTitle>
-        <p>
+        <p className="text-center">
           Please reach out on Discord or to battlecode@mit.edu with the
```

Comment:
Let's replace this with whatever the standard way to link to an email address is! Maybe:
```
<a href="mailto:battlecode@mit.edu?subject=play.battlecode.org%20Bug%20Report&body=This%20is%20the%20email%20body">battlecode@mit.edu</a>
```
where the body is the error message and page name (or some other well-formatted embedding of the error information :smiley:)